.github/workflows/test.yaml

```diff
@@ -140,3 +140,21 @@ jobs:
 
       - name: Perform CodeQL Analysis
         uses: github/codeql-action/analyze@v2
+
+  editorconfig:
```

Comment:
Maybe we can just extend `codeql` with `editorconfig-checker`?

Reply:
codeql takes ~7m to run; editorconfig can be run parallel to that job, and finishes in less than a minute!